toolkit/scripts/containerized-build/containerized-build_clean.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Yes, this has a lot in common with toolchain_clean.sh. I am hesitant to change toolchain_clean.sh because I am not sure if any pipelines/ build systems use it. Are you aware of any potential breakages if I add an argument toolchain_clean.sh (or alternatively, replace toolchain_clean.sh with a common clean_container.sh script that takes container_type as an argument and it use it to clean up toolchain containers)

Reply:
Discussed offline, toolchain_clean.sh is used only on local machines, not in the pipelines. So, going ahead with modifying toolchain_clean.sh to take a container_type arg that would clean up the container_type provided. This would code duplication.